BUILD.gn

```diff
@@ -12,14 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import("//build_overrides/spirv_tools.gni")
```

Comment:
I don't know much about GN. 
How does an external project like Dawn override the settings that we import here?

Reply:
It is purely a convention starteb by Chromium's `build_overrides/build.gni`: each project has a gni in "build_overrides" and does `import("//build_overrides/foo.gni")` first thing in its BUILD.gn. Since `//` is the root of the current GN execution, this will either import SPIRV-Tools's version if in SPIRV-Tools, or the one in `<dawn_source>/build_overrides` if we are in Dawn's source.

Reply:
Ah.  It's like duck-typing for builds.
As long as all the build-overrides file sets the right things for SPIRV-Tools then it all works out. Very nice.